force-app/main/default/classes/CRLP_ResetRollupFieldsQueueable.cls

```diff
@@ -0,0 +1,170 @@
+        if (!updatedRecords.isEmpty()) {
+            CRLP_RollupProcessor_SVC.updateChangedRecordsWithLogging(
+                clearSkewModeFlag(updatedRecords), 
+                jobType
+            );
+        }
+    }
+
+    /**
+     * @description Get updated summary records.
+     * @return List<SObject>
+     */
+    @TestVisible
+    private List<SObject> getUpdatedRecords(List<SObject> parents) {
+        CRLP_RollupProcessingOptions.ProcessingOptions options = new CRLP_RollupProcessingOptions.ProcessingOptions();
+        options.useRollupDefsByParentIdMap = true;
+
+        CRLP_RollupProcessor processor = new CRLP_RollupProcessor()
+            .withRollupType(jobType)
+            .withDetailRecords(new List<SObject>())
+            .withRelatedRecords(new Map<Id, List<SObject>>())
+            .withJobOptions(options);
+
+        List<SObject> updatedRecords = new List<SObject>();
+        for (SObject parent : parents) {
+            SObject updatedRecord = processor.withSummaryRecord(parent).completeRollupForSingleSummaryRecord(parent);
+            if (updatedRecord != null) {
+                updatedRecords.add(updatedRecord);
+            }
+        }
+        return updatedRecords;
+    }
+
+    /**
+     * @description Query summary records by Id.
+     * @return List<SObject>
+     */
+    @TestVisible
+    private List<SObject> getSummaryRecords() {
+        String query = CRLP_Query_SEL.buildObjectQueryForRollup(summaryObjectType) + ' WHERE Id IN :recordIds';
+        List<SObject> parents = Database.query(query);
+        return parents;
+    }
+
+    /**
+     * @description Unconditionally set skew mode flag false.
+     * @return List<SObject>
+     */
+    @TestVisible
+    private List<SObject> clearSkewModeFlag(List<SObject> updatedRecords) {
+        if ((this.summaryObjectType == Account.SObjectType || this.summaryObjectType == Contact.SObjectType)) {
+            for (SObject updatedRecord : updatedRecords) {
+                updatedRecord.put(UTIL_Namespace.StrTokenNSPrefix('CustomizableRollups_UseSkewMode__c'), false);
+            }
+        }
+        return updatedRecords;
+    }
+
+    /**
+     * @description Query parent records to clear.
+     * @return List<Id>
+     */
+    private List<Id> getParentRecordIds() {
+        Set<Id> gausWithChildren = getGAUsWithChildren();
+        String query = 'SELECT Id FROM ' + UTIL_Namespace.StrTokenNSPrefix('General_Accounting_Unit__c') +
+            ' WHERE Id NOT IN :gausWithChildren';
+
+        List<Id> records = new List<Id>();
+        for (SObject record : Database.query(query)) {
+            records.add((Id) record.get('Id'));
+        }
+        return records;
+    }
+
+    /**
+     * @description Query all GAU records with children.
+     * @return Set<Id>
+     */
+    private Set<Id> getGAUsWithChildren() {
+        String gauWithNamespace = UTIL_Namespace.StrTokenNSPrefix('General_Accounting_Unit__c');
```

Comment:
This query was timing out for me in an LDV Sandbox that has 7M Allocation records. The Query Explain gave it a query cost of 4.95541, which is too high.

Reply:
Sounds good.  I'll make that change and prohibit a run with null recordIds.  Thanks!!!

Reply:
@force2b @lparrott null recordIds is now prohibited before queueable is enqueued in CRLP_Batch_Base_Skew.  jobType is restricted to GAU there as well.  

I've left the logic to handle other run types in CRLP_ResetRollupFieldsQueueable, but given where we are in the sprint, it is unrealistic to test other scenarios.  Saving that low priority fix for a second work item.

The costly query attempting to deal with null recordIds has been removed.